project-base/storefront/components/Layout/Header/AutocompleteSearch/AutocompleteSearch.tsx

```diff
@@ -58,7 +59,8 @@ export const AutocompleteSearch: FC = () => {
         }
     }, [searchQueryValue]);
 
-    const isSearchResultsPopupVisible = isSearchResultsPopupOpen && isWithValidSearchQuery && !!searchData;
+    const isSearchResultsPopupVisible =
+        isSearchResultsPopupOpen && isWithValidSearchQuery && (!!searchData || areAutocompleteSearchDataFetching);
```

Comment:
In safari, it behaves very poorly (see video)


https://github.com/shopsys/shopsys/assets/55181354/c29b3bf9-5e31-4e38-be7e-1ee3c797cd24

Reply:
Oh good point, we had same problem on breno, thanks, fixed